packages/fx-core/tests/core/middleware/testAssets/v3Migration/debug/transparent-m365-me/expected/app.local.yml

```diff
@@ -43,9 +43,9 @@ provision:
   - uses: m365Title/acquire # Upload your app to Outlook and the Microsoft 365 app
     with:
       appPackagePath: ./build/appPackage/appPackage.${{TEAMSFX_ENV}}.zip # Relative path to the built app package.
-    # Output: following environment variable will be persisted in current environment's .env file.
-    # M365_TITLE_ID: the id of M365 title
-    # M365_APP_ID: the app id of M365 title
+    writeToEnvironmentFile:
+      titleId: M365_TITLE_ID
+      appId: M365_APP_ID
```

Comment:
@kuojianlu please help review m365Title/acquire. There is no comment for it yet.

Reply:
I think we can add commet for it.

Reply:
Updated it. Thanks.